AdminWebsite/AdminWebsite/ClientApp/src/app/get-audio-file/get-audio-file-request-timedout-warning/get-audio-file-request-timedout-warning.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'app-get-audio-file-request-timedout-warning',
+    templateUrl: './get-audio-file-request-timedout-warning.component.html',
+    styles: ['']
+})
+export class GetAudioFileRequestTimedoutWarningComponent implements OnInit {
```

Comment:
can remove the empty ctor and the OnInit interface if you're not implementing it for this component